x-pack/platform/plugins/shared/ml/public/application/overview/components/content.tsx

```diff
@@ -41,7 +41,7 @@ export const OverviewContent: FC<Props> = ({
 
   useEffect(() => {
     setAnomalyTimelineService(
-      new AnomalyTimelineService(timefilter, uiSettings, mlResultsServiceProvider(mlApi))
+      new AnomalyTimelineService(timefilter, uiSettings, mlResultsServiceProvider(mlApi), mlApi)
```

Comment:
rather than calling `mlResultsServiceProvider` here and passing in the result, you could now just pass in `mlApi` and call `mlResultsServiceProvider` in the constructor